WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductVariationFormEventLogger.swift

```diff
@@ -31,4 +31,12 @@ struct ProductVariationFormEventLogger: ProductFormEventLoggerProtocol {
     func logSubscriptionsTapped() {
         ServiceLocator.analytics.track(event: .Variations.subscriptionsTapped())
     }
+
+    func logSubscriptionsFreeTrialTapped() {
+        ServiceLocator.analytics.track(.productVariationViewSubscriptionFreeTrialTapped)
```

Comment:
Same as the other comment about following the pattern from `func logSubscriptionsTapped()` for improved readability.

Reply:
Handled in https://github.com/woocommerce/woocommerce-ios/commit/20e2f8a8d73d989bd6928e576cc3ace90b59754a